src/htm/engine/Network.hpp

```diff
@@ -117,6 +117,7 @@ class Link;
    *  On errors it throws an exception.
    */
   void configure(const std::string &yaml);
+  std::string getSpecJSON(const std::string &type);
```

Comment:
please add a short doc on the functionality,usage. and what the type param should be.